src/color/color.class.ts

```diff
@@ -223,20 +226,14 @@ export class Color {
       otherColor = new Color(otherColor);
     }
 
-    const result = [],
-      alpha = this.getAlpha(),
+    const [r, g, b, alpha] = this.getSource(),
       otherAlpha = 0.5,
-      source = this.getSource(),
-      otherSource = otherColor.getSource();
-
-    for (let i = 0; i < 3; i++) {
-      result.push(
-        Math.round(source[i] * (1 - otherAlpha) + otherSource[i] * otherAlpha)
```

Comment:
We could expose an option for color interpolation, the `calculate` method is there for such subclassing.
But I want to finalize this PR.
So if you want to propose something fork this branch and start another PR, that's what I do.

Reply:
I would change `ColorAnimation` to accept color string values, that way it can accept any color space.
Then with a color space option it becomes very powerful